projects/knora-ui/src/lib/viewer/values/boolean-value/boolean-value.component.html

```diff
@@ -1,19 +1,25 @@
-<span [formGroup]="form">
-    <mat-checkbox [formControlName]="'booleanValue'" class="value boolValue child-value-component" (change)="onChecked()">
-        {{booleanLabel}}
-    </mat-checkbox>
-    <mat-form-field class="large-field value-component-comment" *ngIf="shouldShowComment">
-        <textarea matInput
-                  cdkTextareaAutosize
-                  #autosize="cdkTextareaAutosize"
-                  cdkAutosizeMinRows="1"
-                  cdkAutosizeMaxRows="6"
-                  [formControlName]="'comment'"
-                  class="comment"
-                  placeholder="Comment"
-                  type="text"
-                  [readonly]="mode === 'read'"
-                  spellcheck="false">
-        </textarea>
-    </mat-form-field>
-</span>
\ No newline at end of file
+<div *ngIf="mode === 'read'; else showForm" class="read-mode-view">
+    <p class="rm-value">{{booleanLabel}}</p>
```

Comment:
I think we should get rid of the extra member `booleanLabel`. The display value should be generated from the `FormControl`'s value.

I created a task for it: https://dasch.myjetbrains.com/youtrack/issue/DSP-205.

Reply:
to discuss with Flavie

Reply:
to be done as part of https://dasch.myjetbrains.com/youtrack/issue/DSP-206